<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="valueLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="methodGroupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>120</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter method</string>
    </property>
    <widget class="QRadioButton" name="meanRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Mean</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="binomialRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Binomial</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sizeGroupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Kernel matrix size</string>
    </property>
    <widget class="QRadioButton" name="size3RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;3x3</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="size5RadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;5x5</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
